src/utils/consent.js

```diff
@@ -8,10 +8,14 @@ export const isConsentGranted = () => {
 
 export const grantConsent = () => {
   const version = appVersion()
-  sendEvent('KaiOSAppFirstRun', 20158341, getAppLanguage(), {
-    app_version: version,
-    app_id: appInstallId()
-  })
+  sendEvent(
+    '/analytics/legacy/kaiosappfirstrun/1.0.0',
+    'eventlogging_KaiOSAppFirstRun', 
+    {
+      app_version: version,
+      app_id: appInstallId()
+    }
+)
```

Comment:
my bad, should be like this

```suggestion
    {
      app_version: version,
      app_id: appInstallId()
    }
)
```
```suggestion
  sendEvent(
    '/analytics/legacy/kaiosappfirstrun/1.0.0',
    'eventlogging_KaiOSAppFirstRun', 
    {
      app_version: version,
      app_id: appInstallId()
    }
  )
```